synthesizer/base_galaxy.py

```diff
@@ -64,6 +64,80 @@ def get_spectra_stellar(
             self.spectra['stellar'] = sed
 
         return sed
+    
+    def get_spectra_incident(
+            self,
+            grid,
+            young=False,
+            old=False,
+            update=True
+    ):
+        """
+        Generate the incident (pure stellar for stars) spectra using the provided Grid
+
+        Args:
+            grid (obj):
+                spectral grid object
+            update (bool):
+                flag for whether to update the `stellar` spectra
+                inside the galaxy object `spectra` dictionary.
+                These are the combined values of young and old.
+            young (bool, float):
+                if not False, specifies age in Myr at which to filter
+                for young star particles
+            old (bool, float):
+                if not False, specifies age in Myr at which to filter
+                for old star particles
+
+        Returns:
+            An Sed object containing the stellar spectra
+        """
+
+        lnu = self.generate_lnu(grid, 'incident', young=young, old=old)
+
+        sed = Sed(grid.lam, lnu)
+
+        if update:
+            self.spectra['incident'] = sed
+
+        return sed
+    
+    def get_spectra_transmitted(
+        self,
+        grid,
+        young=False,
+        old=False,
+        update=True
+    ):
+        """
+        Generate the transmitted spectra using the provided Grid
```

Comment:
This needs to be more detailed. What is transmitted?

Reply:
I believe this is the language used by CLOUDY in both its documentation and outputs, and we decided at some point to honour this naming for the sake of simplicity. But I agree that this should be better explained in the docstrings, and ideally have its own section in the documentation

Reply:
New doc added to documents.

Reply:
Great that we have the docs but this also needs to be in the doc string. Just a brief description, "transmitted, i.e. the incident emission reprocessed by the nebula, ...".